life/updated/02-life.adoc

```diff
@@ -358,137 +358,144 @@ He took me by the hand, and led me by a way I knew not. He made darkness light b
 and has preserved me unto this very day in covenant with Himself.
 "`Everlasting praises and honor be given to His holy name forever,`" says my soul.
 
-You may remember, since you have had an understanding,
-the straits and difficulties the Lord has enabled me to go through,
-though but weak and greatly afflicted with sickness, and very near the grave many times.
-The Lord renewed my strength again,
-to bear a faithful testimony for him and his blessed truth.
-Various straits and hardships has the Lord, my Redeemer, brought me through,
-which when I look back and consider, I am filled with admiration,
+You may remember many of the straits and difficulties the Lord has enabled me to go through,
+though I was but weak and greatly afflicted with sickness, and very near the grave many times.
+The Lord renewed my strength again and again,
+to bear a faithful testimony for Him and His blessed truth.
+Various straits and hardships has the Lord my Redeemer brought me through,
+which when I look back and consider, fille me with admiration
```

Comment:
oops... you said "fille" instead of fill.  And if you're going to work it this way, I think it needs a comma after "which". Like this: which, when I look back and consider, fill me with admiration.